src/html/scope.rs

```diff
@@ -108,20 +108,27 @@ impl<COMP: Component> Scope<COMP> {
     }
 
     /// Send a message to the component
-    pub fn send_message(&self, msg: COMP::Message) {
-        self.update(ComponentUpdate::Message(msg));
+    pub fn send_message<T>(&self, msg: T)
+    where
+        T: Into<COMP::Message>,
+    {
+        self.update(ComponentUpdate::Message(msg.into()));
     }
-
+    
     /// Send a batch of messages to the component
-    pub fn send_message_batch(&self, messages: Vec<COMP::Message>) {
-        self.update(ComponentUpdate::MessageBatch(messages));
+    pub fn send_message_batch<T>(&self, messages: T)
+    where
+        T: Into<Vec<COMP::Message>>,
```

Comment:
Check out this link: https://rust-lang.github.io/api-guidelines/flexibility.html#functions-minimize-assumptions-about-parameters-by-using-generics-c-generic

Let's use `I: IntoIterator<Item = COMP::Message>` for the batch methods

Reply:
@jstarry Thanks for the help!